src/main/java/com/example/gimmegonghakauth/service/CompletedCoursesService.java

```diff
@@ -73,29 +84,19 @@ public void extractExcelFile(MultipartFile file, UserDetails userDetails)
 
             CoursesDomain coursesDomain = coursesDao.findByCourseId(
                 courseId);// 학수번호를 기반으로 Courses 테이블 검색
-            if (coursesDomain == null){
+            if (coursesDomain == null) {
                 continue;
             }
             CompletedCoursesDomain data = CompletedCoursesDomain.builder().userDomain(userDomain)
                 .coursesDomain(coursesDomain).year(year).semester(semester).build();
```

Comment:
```suggestion
                .coursesDomain(coursesDomain)
                .year(year)
                .semester(semester)
                .build();
```
메서드 체이닝에 적절한 줄바꿈을 통해 가독성을 높일 수 있을 것 같습니다!

Reply:
좋은 의견인것 같습니다👍